src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -161,4 +161,21 @@ public int getTotalMessageCount() {
     return results.countEntities(FetchOptions.Builder.withDefaults());
   }
 
+  /**
+   * This is to create an user form data to put in the datastore
+   * TODO: Replace getUser() with UUID and use userChart.getId().toString()
+   */
+  public void storeUserFormData(Tea userChart) {
+    
+    Entity userFormData = new Entity("UserFormData", userChart.getUserName());
```

Comment:
Given that the form asks about "today", not "all time", I'd expect that this entity would have a time component to it. I'd recommend either incorporating the date into the key along with the user, or I'd recommend making both 'user' and 'date' into properties of the entity, relying on the datastore to auto-assign a key (this, though, would mean that you would need to do a query to update the entity for a given user on a given date).